demonstrations/tutorial_shadow_hamiltonian_simulation.metadata.json

```
@@ -0,0 +1,99 @@
+    "categories": [
+        "Quantum Computing",
+        "Quantum Machine Learning"
+    ],
+    "tags": [],
+    "previewImages": [
+        {
+            "type": "thumbnail",
+            "uri": "/_static/demonstration_assets/liesim_extension/thumbnail_liesim_extension.png"
+        },
+        {
+            "type": "large_thumbnail",
+            "uri": "/_static/large_demo_thumbnails/thumbnail_large_liesim_extension.png"
+        }
+    ],
+    "seoDescription": "Shadow Hamiltonian Simulation",
+    "doi": "",
+    "canonicalURL": "/qml/demos/tutorial_shadow_hamiltonian_simulation",
+    "references": [
+        {
+            "id": "Wiersema",
+            "type": "preprint",
+            "title": "Classification of dynamical Lie algebras for translation-invariant 2-local spin systems in one dimension",
+            "authors": "Roeland Wiersema, Efekan Kökcü, Alexander F. Kemper, Bojko N. Bakalov",
+            "year": "2023",
+            "publisher": "",
+            "journal": "",
+            "doi": "10.48550/arXiv.2309.05690",
+            "url": "https://arxiv.org/abs/2309.05690"
+        },
+        {
+            "id": "Goh",
+            "type": "preprint",
+            "title": "Lie-algebraic classical simulations for variational quantum computing",
+            "authors": "Matthew L. Goh, Martin Larocca, Lukasz Cincio, M. Cerezo, Frédéric Sauvage",
+            "year": "2023",
+            "publisher": "",
+            "journal": "",
+            "doi": "10.48550/arXiv.2308.01432",
+            "url": "https://arxiv.org/abs/2308.01432"
+        },
+        {
+            "id": "Somma",
+            "type": "preprint",
+            "title": "Quantum Computation, Complexity, and Many-Body Physics",
+            "authors": "Rolando D. Somma",
+            "year": "2005",
+            "publisher": "",
+            "journal": "",
+            "doi": "10.48550/arXiv.quant-ph/0512209",
+            "url": "https://arxiv.org/abs/quant-ph/0512209"
+        },
+        {
+            "id": "Somma2",
+            "type": "preprint",
+            "title": "Efficient solvability of Hamiltonians and limits on the power of some quantum computational models",
+            "authors": "Rolando Somma, Howard Barnum, Gerardo Ortiz, Emanuel Knill",
+            "year": "2006",
+            "publisher": "",
+            "journal": "",
+            "doi": "10.48550/arXiv.quant-ph/0601030",
+            "url": "https://arxiv.org/abs/quant-ph/0601030"
+        },
+        {
+            "id": "Galitski",
+            "type": "preprint",
+            "title": "Quantum-to-Classical Correspondence and Hubbard-Stratonovich Dynamical Systems, a Lie-Algebraic Approach",
+            "authors": "Victor Galitski",
+            "year": "2010",
+            "publisher": "",
+            "journal": "",
+            "doi": "10.48550/arXiv.1012.2873",
+            "url": "https://arxiv.org/abs/1012.2873"
+        }
```

I suppose we don't need to reference the basedOnPapers paper here?
The Aguilar ref is missing though:
```suggestion
        },
        {
            "id": "Aguilar",
            "type": "preprint",
            "title": "Full classification of Pauli Lie algebras",
            "authors": "Gerard Aguilar, Simon Cichy, Jens Eisert, Lennart Bittel",
            "year": "2024",
            "publisher": "",
            "journal": "",
            "doi": "10.48550/arXiv.2408.00081",
            "url": "https://arxiv.org/abs/2408.00081"
        }
```

---

Yep havent touched the metadata 👍  will add